.circleci/config.yml

```diff
@@ -400,7 +400,7 @@ jobs:
       - copy-from-workspace
       - set-sdk-version
       - run:
-          name: Java Customer Registry Quickstart
+          name: Customer Registry Quickstart (Java/Protobuf SDK)
```

Comment:
maybe better to name them as: "Java/Protobuf SDK: Customer Registry Quickstart"? 

As such, they all get grouped together when displayed in CI?